tests/torch/tensor_statistics/test_tensor_statistics.py

```diff
@@ -176,7 +208,7 @@ def test_collected_statistics(self, collector: Type[TensorStatisticCollectorBase
     @pytest.fixture(params=COLLECTORS)
     def collector_for_interface_test(self, request):
         collector_type = request.param
-        return collector_type(reduction_shapes={(1, )})
+        return collector_type(rs_vs_params={(1, ): {'mode': 'symmetric', 'per_channel': False}})
```

Comment:
`rs_vs_params` could also stand to be better typed. Basically we want to end up in a situation where the string literals such as `'mode'` or `'symmetric'` are only present in the raw config files, or at a very high level where the config is parsed. Everywhere else we should have typed structures and enums.

Reply:
Thanks! I got the idea. `rs_vs_params` are removed from the code at all after making forcing the collector to collect stats for one reduction shape. I update the code.